crates/sui-adapter/src/adapter.rs

```diff
@@ -42,39 +44,64 @@ use sui_types::{
     object::{Data, Object, Owner},
     storage::ChildObjectResolver,
 };
-use sui_verifier::entry_points_verifier::{
-    self, is_tx_context, TxContextKind, RESOLVED_ASCII_STR, RESOLVED_STD_OPTION, RESOLVED_SUI_ID,
-    RESOLVED_UTF8_STR,
+use sui_verifier::{
+    entry_points_verifier::{
+        self, is_tx_context, TxContextKind, RESOLVED_ASCII_STR, RESOLVED_STD_OPTION,
+        RESOLVED_SUI_ID, RESOLVED_UTF8_STR,
+    },
+    verifier,
 };
 
+pub fn default_verifier_config(
+    protocol_config: &ProtocolConfig,
+    is_metered: bool,
+) -> VerifierConfig {
+    let (
+        max_back_edges_per_function,
+        max_back_edges_per_module,
+        max_per_fun_meter_units,
+        max_per_mod_meter_units,
+    ) = if is_metered {
+        // TODO: change these to pull from protocol config
+        (None, None, Some(1000 * 8000), Some(1000 * 8000))
```

Comment:
I understand we''l have to eventually load them from protocol config, but pending that PR, generally prefer setting any numeric static value to a const for readability.

Reply:
> I understand we''l have to eventually load them from protocol config, but pending that PR, generally prefer setting any numeric static value to a const for readability. 

Yup. Not to worry though, I will put the consts in protocol config before the PR is ready for review.